### ISSUE TITLE
Bug(Staging): Jira Server integration 

### ISSUE THREAD

**Issue by MindEvo** [ParabolInc/parabol#7084]
## Jira Server Integration causes the task footer to throw an error

Steps to reproduce:
- Go to team page
- Add Jira server integration
- Refresh page then attempt to access the task footer

Adding the Jira server integration causes the task footer to throw an error message at the user. The error appears regardless of what other integrations are, or are not, in use.

<img src="https://user-images.githubusercontent.com/35351786/185288291-5f14125a-6b21-458e-bf83-50906658b718.png" width="250" />

**Comment by BartoszJarocki** [comment]
Tested locally, the bug is caused by wrong lookup, fixed in #7085

Bug
```
react_devtools_backend.js:4026 Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check your code at TaskIntegrationMenuItem.tsx:35.
```

**Comment by BartoszJarocki** [comment]
Verified on staging, all working now!
